services/keycloak/startup-scripts/00-configure-lagoon.sh

```diff
@@ -1811,6 +1811,15 @@ function configure_keycloak {
     configure_token_exchange
     update_add_env_var_to_project
 
+    # Here we source in any extra configuration
+    echo "Checking for any custom startup scripts"
+    if ls /configure-lagoon-extra/*.sh 1> /dev/null 2>&1; then
+      for FILE in /configure-lagoon-extra/*.sh; do
+        echo "Found $FILE - executing in current shell"
+        source $FILE;
```

Comment:
This should be quoted or it will break if someone puts a space in the name of their script.